.github/workflows/ci.yml

```diff
@@ -28,3 +31,21 @@ jobs:
 
       - name: Tests
         run: ENV=CI bundle exec rake spec
+
+  rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.ruby_version }}
+          bundler-cache: true
+
+      - uses: actions/checkout@v4
+
+      - name: Install gems
+        run: bundle install
```

Comment:
Not needed when using bundler-cache.  But you need to run checkout before setup ruby.